src/MissionCreator.sol

```diff
@@ -64,22 +68,23 @@ contract MissionCreator is Ownable, IERC721Receiver {
         missionTable = MissionTable(_missionTable);
     }
 
-    function createMission(uint256 teamId, address to, string calldata projectUri, uint32 duration, uint256 fundingGoal, bool token, string calldata tokenName, string calldata tokenSymbol, string calldata memo) external returns (uint256) {
+    function createMission(uint256 teamId, address to, string calldata projectUri, uint32 duration, uint256 deadline, uint256 minFundingRequired, uint256 fundingGoal, bool token, string calldata tokenName, string calldata tokenSymbol, string calldata memo) external returns (uint256) {
+
         if(msg.sender != owner()) {
             require(moonDAOTeam.isManager(teamId, msg.sender), "Only a manager of the team or owner of the contract can create a mission.");
         }
 
         address payable toPayable = payable(to);
         address payable moonDAOTreasuryPayable = payable(moonDAOTreasury);
-
         IJBTerminal terminal = IJBTerminal(jbMultiTerminalAddress);
 
         //TODO: Configure ruleset
+        LaunchPadPayHook launchPadPayHook = new LaunchPadPayHook(minFundingRequired, fundingGoal, deadline, jbTerminalStoreAddress, to);
         JBRulesetConfig[] memory rulesetConfigurations = new JBRulesetConfig[](1);
```

Comment:
So I guess this all happens in just one ruleset - nice. I guess the second ruleset can just be the redemption case?

Reply:
yep that's my plan!

Reply:
yep makes a lot of sense